api/lib/libDependencies.js

```diff
@@ -365,5 +373,6 @@ module.exports = {
   createSharedPlan,
   findSharedPlans,
   createVulnerabilitySnapshot,
-  findVulnerabilitySnapshots
+  findVulnerabilitySnapshots,
+  cautionaryAlerts
```

Comment:
I think this file only exports the usecases so the gateway doesn't need to be added here

Reply:
I get the following error  otherwise "'cautionaryAlerts' is assigned a value but never used  no-unused-vars"